src/rules/objectCurlySpacingRule.ts

```diff
@@ -0,0 +1,71 @@
+import * as ts from 'typescript';
+import * as Lint from 'tslint/lib/lint';
+
+const OPTION_ALWAYS = 'always';
+
+export class Rule extends Lint.Rules.AbstractRule {
+  public static FAILURE_STRING = {
+    always: {
+      start: `A space is required after '{'`,
+      end: `A space is required before '}'`
+    },
+    never: {
+      start: `There should be no space after '{'`,
+      end: `There should be no space before '}'`
+    }
+  };
+
+  public apply(sourceFile: ts.SourceFile): Lint.RuleFailure[] {
+    const walker = new BlockSpacingWalker(sourceFile, this.getOptions());
+    return this.applyWithWalker(walker);
+  }
+}
+
+class BlockSpacingWalker extends Lint.RuleWalker {
```

Comment:
Need to change the name. This was a copy/paste mistake from the BlockSpacing rule.